vignettes/userguide.Rmd

```diff
@@ -565,6 +568,11 @@ In this file, `#PBS-*` defines command-line arguments to the `qsub` program.
   in curly braces (`{{ ... }}`), as they are used to fill in the right
   variables.
 
+
+```{r eval=FALSE}
+options(clustermq.scheduler = "PBS")
+```
+
```

Comment:
I would find it easier to understand if "pbs" is referenced above instead (like mentioned in previous comment)